projects/lkft-android/include/lkft-common-actions.jinja2

```diff
@@ -0,0 +1,15 @@
+    - from: inline
+      path: android-boot.yaml
+      name: android-boot
+      repository:
+        metadata:
+          format: Lava-Test Test Definition 1.0
+          name: android-boot
+          description: "android-boot"
+        run:
+          steps:
+            - lava-test-case "android-boot-sleep-30secs" --shell sleep 30
+            - lava-test-case "android-boot-lsusb-v" --shell lsusb -v
+            - lava-test-case "android-boot-boot-completed" --shell "while ! adb shell getprop sys.boot_completed|grep 1; do sleep 2; done"
+            - lava-test-case "android-boot-set-power-stayon" --shell adb shell su 0 svc power stayon true
+            - lava-test-case "android-boot-screencap" --shell adb shell screencap -p /data/local/tmp/screencap.png
```

Comment:
What makes this file specific to lkft-android? and not just android?

Reply:
this is just some commands that run after the boot, other cases might not  need to run or need to run all of them, so it's better to have these commands for project only instead of putting them in the common place.